code/controllers/configuration.dm

```diff
@@ -177,6 +177,7 @@
 	var/disable_lobby_music = 0 // Disables the lobby music
 	var/disable_cid_warn_popup = 0 //disables the annoying "You have already logged in this round, disconnect or be banned" popup, because it annoys the shit out of me when testing.
 
+	var/use_iterative_explosions //Defines whether the server uses recursive or circular explosions.

```

Comment:
pretty sure by default its a null reference, which is null and will not match the if check, but i'm not gonna argue with that either way i also prefer 0, not sure why i didnt do it in the first place XD.